src/WingetCreateCLI/DscResources/microsoft.winget-create.settings.dsc.resource.json

```diff
@@ -0,0 +1,62 @@
+{
+    "$schema": "https://raw.githubusercontent.com/PowerShell/DSC/main/schemas/2024/04/bundled/resource/manifest.json",
+    "description": "Allows management of settings state via the DSC v3 command line interface protocol.",
+    "export": {
+        "executable": "wingetcreate",
+        "input": "stdin",
+        "args": [
+            "dsc",
+            "settings",
+            "--export"
+        ]
+    },
+    "get": {
+        "executable": "wingetcreate",
+        "input": "stdin",
+        "args": [
+            "dsc",
+            "settings",
+            "--get"
+        ]
+    },
+    "schema": {
+        "command": {
+            "executable": "wingetcreate",
+            "args": [
+                "dsc",
+                "settings",
+                "--schema"
+            ]
+        }
+    },
+    "set": {
+        "executable": "wingetcreate",
+        "implementsPretest": true,
+        "return": "stateAndDiff",
+        "args": [
+            "dsc",
+            "settings",
+            {
+                "jsonInputArg": "--set",
+                "mandatory": true
+            }
+        ]
+    },
+    "test": {
+        "executable": "wingetcreate",
+        "return": "stateAndDiff",
+        "args": [
+            "dsc",
+            "settings",
+            {
+                "jsonInputArg": "--test",
+                "mandatory": true
+            }
+        ]
+    },
+    "tags": [
+        "WinGetCreate"
+    ],
+    "type": "Microsoft.WinGetCreate/Settings",
+    "version": "1.9.0"

```

Comment:
I just realized winget-create has not been updated to latest since 1.9...

Reply:
Update version to 1.10.0 after the PR above was merged :)